### ISSUE TITLE
[XamarinProfiler-MAC]: Accessibility: Color Contrast: MAS1.4.3: "Common Options", "Sampling", "Allocations" and grey text in the window is having color contrast ratio of 4.5:1

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#414]
### Steps to Reproduce

1. Follow the pre-requisite to launch the Xamarin profiler.
2. Select the Available instruments to "Allocations/Time Profiler" and invoke next button. This will invokes Common options window.
3. Verify the Color contrast ratio of all the text in grey(in both Allocation/Time Profiler) in common options window.

### Pre-Requisite:
Install Xamarin Profiler.
Launch Visual Studio.
Create a project and build the solution.
Navigate to Run Menu Tab and Invoke it. Navigate to "Start Profiling" option and invoke it.
This will invokes "Xamarin Profiler" application.

### Expected Behavior
Color contrast ratio should be minimum 4.5:1
### Actual Behavior
Color contrast ratio is less than 4.5:1 i.e, 2.7:1
See the screenshots attached
### User Impact
Low vision user will feel difficult in seeing the text

### Screenshot
![2 common options](https://user-images.githubusercontent.com/22956130/41590036-aa7cc212-73d3-11e8-8e78-c16f5e8b1e14.png)

### Environment
macOS High Sierra OSX version: 10.13.4
Visual Studio for Mac Enterprise version: 7.5.2(build 40)
Xamarin Profiler version: 1.6.3-400

**Comment by WiproPSAccTest** [comment]
Please add the following labels to this bug:
A11yMAS
Accessibility
MAS1.4.3
XamarinProfiler-MAC
Wipro
